File: C++/Daily Flash/week1/Day6/code_C++/prog3.cpp
/*
 * Program 3 : Write a Program to print First 50 Odd Numbers .
Output: 1 3 5 . . . 99
*
*/

#include <iostream>

int main() {

	std::cout << "First 50 odd Numbers: " << std::endl;
	int num =1;

	for(int i=1; i<= 50; i++){
		std::cout << num << "  ";
		num +=2;
	}
	std::cout << std::endl;
}

File: JAVA/Daily Flash Codes/Week2/Day5_Jan24/code/prog3.java
/*
 * Program 3: Write a Program to print Cubes and Squares of all Odd numbers
ranges between given input number.
Input: 10
Output:
Cube of 1 : 1 and Square of 1 :1
Cube of 3 : 27 and Square of 3 :9
.
.
Cube of 9 : 729 and Square of 9 : 81*/


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		int num;
		System.out.println("Enter num: ");
		num = Integer.parseInt(in.readLine());
		
		int count = 0;
		for(int i=1; i<=num; i++) {
			System.out.println("cube of" + i + ": " + (i*i*i) + "square of " + i +": "+ (i*i)) ;
			count++;
		}
		System.out.println();	
	}

}
